src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.Windows.cs

```diff
@@ -32,8 +33,36 @@ namespace Uno.Toolkit.UI
 {
 	public partial class ExtendedSplashScreen
 	{
+		private static bool IsBrowser { get; } =
+				// Origin of the value : https://github.com/mono/mono/blob/a65055dbdf280004c56036a5d6dde6bec9e42436/mcs/class/corlib/System.Runtime.InteropServices.RuntimeInformation/RuntimeInformation.cs#L115
+				RuntimeInformation.IsOSPlatform(OSPlatform.Create("WEBASSEMBLY")) // Legacy Value (Bootstrapper 1.2.0-dev.29 or earlier).
+				|| RuntimeInformation.IsOSPlatform(OSPlatform.Create("BROWSER"));
+
 		private async Task<FrameworkElement?> GetNativeSplashScreen(SplashScreen? splashScreen)
 		{
+#if WINDOWS_UWP || WINDOWS
```

Comment:
File name is `Windows.cs`. If this file is applicable to Wasm/Skia, then the file name is inaccurate.

Reply:
Renaming the file makes it hard to review the changes - Happy to review the file once we're comfortable with the rest of the changes

Reply:
Updated